indico/modules/events/timetable/client/js/Timetable.module.scss

```diff
@@ -79,6 +81,17 @@
 
       &.selected {
         filter: opacity(0.9) brightness(0.95);
+        box-shadow: 4px 0 18px -2px $indico-blue;
+        margin-left: 4px;
```

Comment:
```suggestion
        box-shadow: -0.1em 0 1em 0.2em $indico-blue;
        margin-left: 0.2em;
```

just to fill up that gap on the left a little better
![image](https://github.com/duartegalvao/indico/assets/27357203/8fe8ecea-111f-4caf-b8fa-6c879a5ffb55)